packages/hatchify-core/src/types/schemaV2.ts

```diff
@@ -1,7 +1,9 @@
-export type UserValue = number | string | Date | object | null | undefined
-
 export type ValueInRequest = number | string | Date | object | null | undefined
 
+export type UserValue = ValueInRequest
```

Comment:
this is just an alias for `ValueInRequest` at the moment, but made a separate type just in case these vary in the future; for example I think `ValueInRequest` should not have `Date` but `UserValue` can have it